### ISSUE TITLE
How to use system properties in Maven plugin?

### ISSUE THREAD

**Issue by morettileo** [smallrye/smallrye-open-api#1822]
With reference to #1063, I tried to specify some properties but they are ignored.

```
<plugin>
  <artifactId>smallrye-open-api-maven-plugin</artifactId>
  <groupId>io.smallrye</groupId>
  <version>3.10.0</version>
  <executions>
	  <execution>
		  <id>generate-openapi</id>
		  <phase>process-classes</phase>
		  <goals>
			  <goal>generate-schema</goal>
		  </goals>
		  <configuration>
			  <scanDependenciesDisable>true</scanDependenciesDisable>
			  <systemPropertyVariables>
				  <mp.openapi.extensions.smallrye.defaultConsumes>*/*</mp.openapi.extensions.smallrye.defaultConsumes>
				  <mp.openapi.extensions.smallrye.auto-inheritance>BOTH</mp.openapi.extensions.smallrye.auto-inheritance>
			  </systemPropertyVariables>
		  </configuration>
	  </execution>
  </executions>
</plugin>
```

Neither mp.openapi.extensions.smallrye.defaultConsumes or mp.openapi.extensions.smallrye.auto-inheritance has effect on generation.

**Comment by MikeEdgar** [comment]
I think this is a bug that was fixed with #1774, which will be in the next release soon.

**Comment by codespearhead** [comment]
@morettileo PR #1774 is available in [v3.11.0](https://github.com/smallrye/smallrye-open-api/releases/tag/3.11.0#:~:text=1774), which was released two days ago.

Is this problem solved?

**Comment by Postremus** [comment]
Hey @MikeEdgar 
I just tested this. With 3.11 systemProperties work.

**Comment by MikeEdgar** [comment]
Thanks @Postremus 

Fixed by #1774
